src/layouts/Authenticated.vue

```diff
@@ -446,29 +447,88 @@ export default defineComponent({
       }
     });
 
+    /**
+     * Order of priority of incident id (descending):
+     * - Route path parameter (incidents/123/work)
+     * - User states current incident id
+     * - Most recent incident they have access too
+     */
+    async function resolveInitialIncidentId() {
+      const u = currentUser.value;
+      const incidentIdFromRoute = route.params?.incident_id;
+      const recentIncidentIdFromState = u?.states?.incident as number;
+      const approvedIncidentId = u?.approved_incidents?.[0] as number;
+      debug('incidentId sources %o', {
+        incidentIdFromRoute,
+        recentIncidentIdFromState,
+        approvedIncidentId,
+      });
+      let incidentId =
+        incidentIdFromRoute ?? recentIncidentIdFromState ?? approvedIncidentId;
+      if (!incidentId) {
+        debug('Cant resolve incident id from route or user states', incidentId);
+        // get most recent incident
+        const _recentIncidents = await Incident.api().get(
+          `/incidents?fields=${incidentFieldsStr.value}&limit=1&sort=-start_at`,
+          { dataKey: 'results' },
+        );
+        const incident = Incident.query().orderBy('id', 'desc').first();
```

Comment:
3.) Again: reactively.
```suggestion
			  const latestIncident = computed(() => Incident.query().orderBy(..).first());
			  const currentIncidentState = computed(() => userStates.currentIncidentId);
			  const hasIncident = computedEager(() => Boolean(currentIncidentState.value) || Boolean(currentIncidentState.value));
			  whenever(logicNot(hasIncident), async () => await <fetch most recent incident and load into store>);
			  // ...etc
```